package.json

```diff
@@ -832,72 +832,13 @@
                     "description": "Indent size for formatting.",
-                    "scope": "window"
-                },
-                "julia.format.indents": {
-                    "type": "boolean",
-                    "default": true,
-                    "description": "Format file indents.",
-                    "scope": "window"
-                },
-                "julia.format.ops": {
-                    "type": "boolean",
-                    "default": true,
-                    "description": "Format whitespace around operators.",
-                    "scope": "window"
-                },
-                "julia.format.tuples": {
-                    "type": "boolean",
-                    "default": true,
-                    "description": "Format tuples.",
-                    "scope": "window"
-                },
-                "julia.format.curly": {
-                    "type": "boolean",
-                    "default": true,
-                    "description": "Format braces.",
-                    "scope": "window"
-                },
-                "julia.format.calls": {
-                    "type": "boolean",
-                    "default": true,
-                    "description": "Format function calls.",
-                    "scope": "window"
-                },
-                "julia.format.iterOps": {
-                    "type": "boolean",
-                    "default": true,
-                    "description": "Format loop iterators.",
-                    "scope": "window"
-                },
-                "julia.format.comments": {
-                    "type": "boolean",
-                    "default": true,
-                    "description": "Format comments.",
-                    "scope": "window"
-                },
-                "julia.format.docs": {
-                    "type": "boolean",
-                    "default": true,
-                    "description": "Format inline documentation.",
-                    "scope": "window"
-                },
-                "julia.format.keywords": {
-                    "type": "bool",
-                    "default": true,
-                    "description": "Ensure single spacing following keywords.",
-                    "scope": "window"
+                    "scope": "application"
                 },
-                "julia.format.kwarg": {
-                    "type": "string",
-                    "default": "none",
-                    "description": "Format whitespace around function keyword arguments.",
-                    "enum": [
-                        "none",
-                        "single",
-                        "off"
-                    ],
-                    "scope": "window"
+                "julia.format.margin": {
+                    "type": "integer",
+                    "default": 100,
+                    "description": "Maximum line length for formatting.",
+                    "scope": "application"
```

Comment:
I think we had said that the default should be to never do line-breaks, right? So that would suggest we either need a different default here, although I would much prefer to just move all config out of VS Code for formatting.